templates/workloads/partials/cf/env-controller-action.yml

```diff
@@ -0,0 +1,15 @@
+Metadata:
+  'aws:copilot:description': "Update your environment's shared resources"
+Type: Custom::EnvControllerFunction
+Properties:
+  ServiceToken: !GetAtt EnvControllerFunction.Arn
+  Workload: !Ref WorkloadName
+  EnvStack: !Sub '${AppName}-${EnvName}'
+  Parameters:
+    {{- if eq .Network.SubnetsType "PrivateSubnets"}}
```

Comment:
I wonder how we can do this in a scalable way as `'EFSWorkloads'` will be added soon.

Maybe we should have a `{{ .EnvControllerParameters }}` method? 

```go
func (o *WorkloadOpts) EnvControllerParameters []string {
   parameters := []string{} // make it non-nil with a size of 0.
   if o.WorkloadType == manifest.LoadBalancedWebService {
       parameters = append(parameters, "ALBWorkloads")
   }
   // if SubnetsType is private, then add
   // if Storage is managed, then add
   return parameters
}
```

Reply:
So then I can just get rid of the two partials for env controller action, correct?